data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/discovery/DnsPeerListProvider.java

```diff
@@ -38,9 +39,10 @@ public DnsPeerListProvider(final DnsAddressEndpointGroup endpointGroup) {
             throw new RuntimeException("Caught exception while querying DNS", e);
         }
 
+        pluginMetrics.gauge(PEER_ENDPOINTS, endpointGroup, group -> group.endpoints().size());
```

Comment:
Where is `PEER_ENDPOINTS` defined? I don't see it in this class or imported

Reply:
This is defined here in the interface. https://github.com/opensearch-project/data-prepper/blob/8eac61a23280cdcfc484afb292836ca729efd0af/data-prepper-core/src/main/java/org/opensearch/dataprepper/peerforwarder/discovery/PeerListProvider.java#L19

Reply:
Ooh, got it. Thanks for the code pointer